tests/moduletemplates/moduletemplate_template_operator_v1_regular.yaml

```diff
@@ -5,8 +5,12 @@ metadata:
   namespace: kcp-system
```

Comment:
To reviewer:

All these moduletemplates are include the managed resource `Managed` from this [PR](https://github.com/kyma-project/template-operator/pull/162), you can of cause download the content from oci registry to verify.